### PR TITLE
added jax numpy logic_or

### LINKED ISSUE #10312: logical_or

### PR DESCRIPTION
**Pull request by xc720**
added jax numpy logic_or
Closes #10312

### CODE REVIEW

File: ivy/functional/frontends/jax/numpy/logic.py [hunk 1]
```diff
@@ -127,7 +127,6 @@ def any(a, axis=None, out=None, keepdims=False, *, where=None):
 
 alltrue = all
 
-
```
Comment (kurshakuz) [inline ivy/functional/frontends/jax/numpy/logic.py:130]: Please remove that line change

File: ivy/functional/frontends/jax/numpy/logic.py [hunk 2]
```diff
@@ -157,3 +156,8 @@ def isinf(x, /):
 def isclose(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
     a, b = promote_jax_arrays(a, b)
     return ivy.isclose(a, b, rtol=rtol, atol=atol, equal_nan=equal_nan)
+
+
+@to_ivy_arrays_and_back
+def logical_not(x, /):
```
Comment (kurshakuz) [inline ivy/functional/frontends/jax/numpy/logic.py:162]: Can you place your function in the alphabetical order?

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_logic.py [hunk 3]
```diff
@@ -20,13 +20,13 @@
...
-    frontend,
-    test_flags,
+        *,
+        dtype_and_input,
+        equal_nan,
+        on_device,
+        fn_tree,
+        frontend,
+        test_flags,
 ):
```
Comment (kurshakuz) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_logic.py:30]: Please don't change lines that are not specific to your PR

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_logic.py [hunk 4]
```diff
@@ -642,13 +642,13 @@ def test_jax_numpy_isinf(
...
-    fn_tree,
-    frontend,
-    test_flags,
+        *,
+        dtype_and_input,
+        equal_nan,
+        on_device,
+        fn_tree,
+        frontend,
+        test_flags,
```
Comment (kurshakuz) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_logic.py:651]: Similarly for all the function changes above

**Review by kurshakuz — CHANGES_REQUESTED** [review]
Hello @xc720! Can you please check my requested changes and apply them before we can start checking the implementaion? Thanks 🙂

File: ivy/functional/frontends/jax/numpy/logic.py (hunk 2 above)
Comment (xc720) [reply ivy/functional/frontends/jax/numpy/logic.py]: yes, I put it below the logical_and function now

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_logic.py (hunk 4 above)
Comment (xc720) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_logic.py]: I updated the test_jax_numpy_logic.py to the newest version and found isfinite from line 607 to 631 was recently added to master so I kept it. And other functions are the same as the newest version. Thank you! :) Please let me know immediately if any other changes are needed!

**Review by kurshakuz — CHANGES_REQUESTED** [review]
Hello @xc720! logical_not is already implemented and added to the repository. Your issue tracks the logical_or implemmentation. Can you remove your function and implement logical_or as suggested by your initial issue?

**Comment by kurshakuz** [comment]
Your tracked issue https://github.com/unifyai/ivy/issues/10312 references implementation of logical_or

**Comment by xc720** [comment]
> Your tracked issue #10312 references implementation of logical_or

Yes, I don't know why it is changed somehow, I will implement it real quick

**Comment by xc720** [comment]
> > Your tracked issue #10312 references implementation of logical_or
> 
> Yes, I don't know why it is changed somehow, I will implement it real quick

Hello, I think now it's implemented, please have a look.

**Comment by kurshakuz** [comment]
Looking good, can you finally run the test function locally and post the result here?

**Comment by xc720** [comment]
> Looking good, can you finally run the test function locally and post the result here?

Yes, I think it works all right, all tests passed.
image

**Review by kurshakuz — APPROVED** [review]
LGTM!

**Pull request merged**